CMakeLists.txt

```diff
@@ -116,16 +141,21 @@ if (UNIX)
     install(FILES       "${CMAKE_CURRENT_BINARY_DIR}/libgraphqlparser.pc"
             DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/pkgconfig")
   endif()
-endif()
 
-IF (test)
-  ADD_SUBDIRECTORY(test)
+  IF (test)
+    ADD_SUBDIRECTORY(test)
 
-  if(UNIX)
-    # setup valgrind
-    ADD_CUSTOM_TARGET(memcheck
-      valgrind --leak-check=full --suppressions=./test/valgrind.supp --dsymutil=yes --error-exitcode=1 ./test/runTests  >/dev/null
-    )
-  endif()
+    if(UNIX)
```

Comment:
You deleted the `endif()` above on what used to be line 119, so this `IF (test)` is already nested within `if(UNIX)`. Did you mean to delete it? I think we should support running tests on Windows if we're going to support Windows. I think we should support running tests on Windows...

Reply:
I'm not sure `valgrind` is available on Windows, I'll check. It might be better to nest the `if(UNIX)` inside of `if(test)` though, so if there are tests that run equally well on Windows (or that can be made to run there) then they can go outside of the platform check.